packages/components/src/components/content-switcher/_content-switcher.scss

```diff
@@ -51,15 +51,34 @@
     color: $text-secondary;
     text-align: left;
     text-decoration: none;
-    transition: all $duration--fast-01 motion(standard, productive);
+    transition: all $duration--moderate-01 motion(standard, productive);
     white-space: nowrap;
 
+    &::after:not(&:disabled) {
```

Comment:
```suggestion
    &::after {
```

Reply:
To cover the case of a disabled content switcher, add a new rule like so:

```scss
&:disabled::after {
  display: none;
}
```

Reply:
Awesome! For some reason my old changes worked with the not selector, but magically it stopped working. But thank you for this @janhassel !